pkg/virt-handler/virt-chroot/virt-chroot.go

```diff
@@ -28,20 +30,27 @@ import (
 )
 
 const (
-	binaryPath     = "/usr/bin/virt-chroot"
-	mountNamespace = "/proc/1/ns/mnt"
+	binaryPath = "/usr/bin/virt-chroot"
 )
 
+// GetHostNamespacePath - return the parent process
+// this is needed to handle the case kubernetes and kubevirt
+// running inside a container and the mount namespace is different
+// from the host/node.
+func GetHostNamespacePath() string {
```

Comment:
super nit: I would move this under `pkg/virt-handler/isolation/process.go`

Reply:
Sure.

Reply:
@alicefr I tried but it causes the problem of recursive importing issue, so for now I just leave this as is. At least is together with the original functions.